src/rcms/fm/app/level1/HCALlevelOneEventHandler.java

```diff
@@ -266,7 +266,7 @@ public void initAction(Object obj) throws UserActionException {
         functionManager.getHCALparameterSet().put(new FunctionManagerParameter<StringT>("HCAL_RUN_TYPE",new StringT(RunType)));
         functionManager.getHCALparameterSet().put(new FunctionManagerParameter<StringT>("GLOBAL_CONF_KEY",new StringT(GlobalConfKey)));
 
-        // RUN_CONFIG_SELECTED = LocalRunKey; CFGSNIPPET_KEY_SELECTED = MasterSnippet file of LocalRunKey
+        // CFGSNIPPET_KEY_SELECTED = LocalRunKey;  RUN_CONFIG_SELECTED= MasterSnippet file of LocalRunKey
```

Comment:
The previous comment was wrong. Fixing this in the code to avoid further confusion.